test/functional/apps/discover/group3/_request_counts.ts

```diff
@@ -54,30 +53,34 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     const expectSearchCount = async (type: 'ese' | 'esql', searchCount: number) => {
-      await retry.try(async () => {
-        if (searchCount === 0) {
-          await browser.execute(async () => {
-            performance.clearResourceTimings();
-          });
-        }
-        await waitForLoadingToFinish();
-        const endpoint = type === 'esql' ? `${type}_async` : type;
-        const requests = await browser.execute(() =>
-          performance
-            .getEntries()
-            .filter((entry: any) => ['fetch', 'xmlhttprequest'].includes(entry.initiatorType))
-        );
+      await retry.tryWithRetries(
+        `expect ${type} request to match count ${searchCount}`,
```

Comment:
```suggestion
        `expect ${type} request to match count ${searchCount}, but received ${count}`,
```
Nit: If these fail in the future, the logs won't include the received count. Not a big deal, but could be worth defining `count` above so we can log it here.

Reply:
I think `tryWithRetries` works similar to `try`. it's using `retryForSuccess` under the hood:

https://github.com/elastic/kibana/blob/9d22e8c6a8d92449055fd99b67b745d1995b5107/packages/kbn-ftr-common-functional-services/services/retry/retry.ts#L113

and since the expect in the try function hasn't change, there should be no problem here?

I think it's `waitFor` that works differently? 
https://github.com/elastic/kibana/blob/9d22e8c6a8d92449055fd99b67b745d1995b5107/packages/kbn-ftr-common-functional-services/services/retry/retry.ts#L83

Reply:
I was suggesting to include `count` so it would still be logged when the test fails, but I think I may have misunderstood what the passed string if for. I was thinking it would override the original error message, e.g. `expected 4 to equal 3`, but I see now the param is called `description`, so maybe it's used for something else and the original error is still logged on failure. I think we're all good here though and can merge this.

Reply:
yes, it's just describing the loop, similar like in the `waitFor` function, no changes in the error message